<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCTemp</class>
 <widget class="QWidget" name="FormCTemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="formAlignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="cTemp0Label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;C&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="cTemp0">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cTemp1Label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;C&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="cTemp1">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cTemp2Label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;C&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="cTemp2">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="cTemp3Label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;C&lt;span style=&quot; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="cTemp3">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
